src/Utility.js

```diff
@@ -4,77 +4,29 @@ import activityData from './data/sleep';
 
 
 class Utility {
-  //needs to take in date info from somewhere
   constructor(dataSet) {
     this.dataSet = dataSet;
-    // this.isSleep = dataSet === sleepData;
-    // this.isHydration = dataSet === hydrationData;
-    // this.isActivity = dataSet === activityData;
   }
 
   calculateDailyAverage(id, relevantData) {
-    // console.log(this.dataSet, relevantData);
-    // returns relevantData(sleepQUality etc. daily average for users data)
     let perDayAmount = this.dataSet.filter((data) => id === data.userID);
     return perDayAmount.reduce((sumSoFar, data) => {
       return sumSoFar += data[relevantData];
     }, 0) / perDayAmount.length;
   }
 
-  displayDailyData(id, date, relevantData) { // returns single datapoint for a user on a day
+  displayDailyData(id, date, relevantData) {
```

Comment:
Good call getting rid of some of these comments. I think we'll have to go through and look for more throughout